src/App.tsx

```diff
@@ -28,129 +67,79 @@ export const App: React.FC = () => {
-            </label>
-
-            <span className="todo__title">Todo is being saved now</span>
-            <button type="button" className="todo__remove">×</button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {todos.length > 0 && (
+          <>
+            <TodoList todos={filteredTodos} />
+            <footer className="todoapp__footer">
+              <span className="todo-count">
+                {`${todos.length} items left`}
+              </span>
+
+              <nav className="filter">
+                <a
+                  href="#/"
+                  className={cn('filter__link', {
+                    selected: filter === Filter.ALL,
+                  })}
+                  onClick={(event) => {
+                    event.preventDefault();
+                    setFilter(Filter.ALL);
+                  }}
+                >
+                  All
+                </a>
+
+                <a
+                  href="#/active"
+                  className={cn('filter__link', {
+                    selected: filter === Filter.ACTIVE,
+                  })}
+                  onClick={(event) => {
+                    event.preventDefault();
+                    setFilter(Filter.ACTIVE);
+                  }}
+                >
+                  Active
+                </a>
+
+                <a
+                  href="#/completed"
+                  className={cn('filter__link', {
+                    selected: filter === Filter.COMPLETED,
+                  })}
+                  onClick={(event) => {
+                    event.preventDefault();
+                    setFilter(Filter.COMPLETED);
+                  }}
+                >
+                  Completed
+                </a>
+              </nav>
+
+              {/* don't show this button if there are no completed todos */}
+              {filteredTodos.some((todo) => todo.completed) && (
```

Comment:
```suggestion
              {filteredTodos.some((todo) => todo.completed) && (
```
would be great to move it to the variable and add this variable here